tests/libtest.sh

```diff
@@ -1,5 +1,18 @@
 #!/bin/bash
 
+# Test if config exists.
+config_exists(){
+  local config_dir="/var/lib/container-storage-setup"
```

Comment:
I am fine with whatever @rhvgoyal wants, I just don't see defining the same constant in multiple functions, when it does not change especially for tests.